packages/terra-data-grid/src/subcomponents/ColumnHeaderCell.jsx

```diff
@@ -159,16 +159,24 @@ const ColumnHeaderCell = (props) => {
 
   const columnContext = useContext(ColumnContext);
   const columnHeaderCellRef = useRef();
+  const columnHeaderCellButtonRef = useRef();
 
   const [isResizeHandleActive, setResizeHandleActive] = useState(false);
 
+  const columnHeaderFocusArea = useCallback(() => (columnHeaderCellButtonRef.current ? columnHeaderCellButtonRef.current : columnHeaderCellRef.current), []);
```

Comment:
Do we really need a useCallback here?  This should really just be a simple or statement to obtain this value, when needed.

Reply:
Why can't we just use a ref variable?